src/test/java/uk/gov/pay/ledger/transaction/dao/TransactionDaoSearchIT.java

```diff
@@ -843,13 +843,56 @@ public void searchTransactionsByCursor() {
                 .withCreatedDate(now(ZoneOffset.UTC).minusDays(1))
                 .insert(rule.getJdbi());
 
-        searchParams.setAccountId("1");
+        searchParams.setAccountIds(List.of("1"));
```

Comment:
Your country thanks you for your service.

Reply:
lol